TCAT/InformationViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  InformationViewController.swift
+//  TCAT
+//
+//  Created by Ji Hwan Seung on 19/11/2017.
+//  Copyright © 2017 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+class InformationViewController: UIViewController {
+    
+    var titleLabel = UILabel()
+    var appDevImage = UIImageView()
+    var appDevTitle = UILabel()
+    var descriptionLabel = UILabel()
+    var tcatQuoteText = UILabel()
+    var tcatImage = UIImageView()
+    var sendFeedbackButton = UIButton()
+    var visitWebsiteButton = UIButton()
+    var backButton = UIButton(type: .system)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = "About Us"
+        
+        view.backgroundColor = UIColor.tableBackgroundColor
+        navigationController?.navigationBar.tintColor = .primaryTextColor
+        
+        view.addSubview(titleLabel)
```

Comment:
adding a subview is the last step. You should only add it once you've formatted it how you like. It's on the order of ms, but I feel like it's good practice

Reply:
But to use constraints, I have to add them to the view before setting them?

Reply:
Oh right, using snapkit and all